Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIWebNavigationCocoa.mm

```diff
@@ -98,6 +98,12 @@ static WebExtensionFrameParameters frameParametersForFrame(_WKFrameTreeNode *fra
     }
 
     [webView _frames:makeBlockPtr([this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler), tab, frameIdentifier](_WKFrameTreeNode *mainFrame) mutable {
+        if (!mainFrame) {
```

Comment:
I don't think this pointer will be null. If the mainframe does not exist it looks like the main frame info will just contain empty data.

Reply:
gotcha. i'll update the check. thanks!